storybookjs/storybook
# Build: Upgrade to Yarn 3

## What I did

 - Upgraded to Yarn 3 (-> specific version, see https://github.com/storybookjs/storybook/pull/15682#issuecomment-887370903)
 - Add a missing dep in `@storybook/core-common`
 - Remove unneeded `@types` dependency
 - Simplify Yarn config in `web-components-kitchen-sink`

## How to test

- CI should be 🟢 
- Checkout this branch and try to run: 
      - `yarn bootstrap --core`
      - `cd examples/official-storybook && yarn && yarn storybook` 
      - `cd ../web-components-kitchen-sink && yarn && yarn storybook`
